aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -1979,6 +1998,116 @@ Tensor cholesky_inverse(const Tensor &input, bool upper) {
   return result;
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ linalg.det ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Could you please move this back to `aten/src/ATen/native/LinearAlgebra.cpp`? That way the history of changes is better preserved and it's easier to compare what's being modified.